File: src/main/java/br/com/logique/hibernatehistory/dto/History.java
package br.com.logique.hibernatehistory.dto;

import lombok.Builder;
import lombok.Data;

/**
 * @author victor.
 */
@Data
@Builder
public class History {

    private Long revision;
    private Integer revisionType;
    private String date;
    private String description;
    private String author;
    private Object object;

}

File: web/src/app/model/history.ts
/**
 * Created by victor on 01/09/17.
 */
export class History {
  revision : number = null;
  revisionType : number = null;
  date : Date = null;
  description : String = "";
  author : String = "";
  object : Object = null;
}

export const RevisionType = {
  0: "INSERÇÃO",
  1: "EDIÇÃO",
  2: "EXCLUSÃO"
}
